### DISCUSSION TITLE
Getting error while creating user

### DISCUSSION THREAD

**Discussion by dikshauat** [Sunbird-Ed/Community#218] [General Discussion]
Hi @keshavprasadms, while cerating new user we are getting the following error on Postman

![image](https://user-images.githubusercontent.com/112255486/194522301-1f62dc5b-f96e-4abd-b9d6-00b7c20f9874.png)

Kindly help us resolving the issue

**Comment by beepdot** [comment]
Please check nginx logs to understand what is happening. One / more of your services might be down.
